src/pybind/mgr/dashboard/frontend/src/app/ceph/smb/smb-cluster-list/smb-cluster-list.component.ts

```diff
@@ -54,6 +61,16 @@ export class SmbClusterListComponent extends ListWithDetails implements OnInit {
         flexGrow: 2
       }
     ];
+    this.tableActions = [
+      {
+        name: `${this.actionLabels.CREATE} Cluster`,
```

Comment:
Just realized,  we usually do not specify the resource, just the action
```suggestion
        name: `${this.actionLabels.CREATE}`,
```